core/src/main/resources/hudson/model/BooleanParameterDefinition/config_it.properties

```diff
@@ -1,7 +1,7 @@
 # The MIT License
 #
 # Italian localization plugin for Jenkins
-# Copyright � 2020 Alessandro Menti
+# Copyright � 2020 Alessandro Menti
```

Comment:
🤦

Reply:
Italian localization files have a messed up encoding, better to not attempt editing.

Reply:
should be fixed in the most recent update.